xarray/core/missing.py

```diff
@@ -437,6 +437,16 @@ def bfill(arr, dim=None, limit=None):
     ).transpose(*arr.dims)
 
 
+def _import_interpolant(interpolant, method):
+    """Import interpolant from scipy.interpolate."""
+    try:
+        from scipy import interpolate
+
+        return getattr(interpolate, interpolant)
+    except ImportError:
+        raise ImportError(f"Interpolation with method {method} requires scipy")
```

Comment:
can we do something like `raise ImportError(...) from e` here? That way we don't drop the additional information the original `ImportError` might have, which is useful when the `ImportError` is not a `ModuleNotFoundError`.

Reply:
Done.

Reply:
great, thanks.